grails-app/conf/spring/resources.groovy

```diff
@@ -0,0 +1,10 @@
+package spring
+
+import util.CustomMessageSource
+
+beans = {
+    messageSource(CustomMessageSource) {
```

Comment:
Can you add a comment to explain the purpose of this bean as well as any other details regarding the implementation that might be helpful / pertinent for future us?